Hero maciek

Smutnyyy opened the pull request:
Hero part. 
Background shown as image. Main text in the centre with decorative accent

MHase commented on styles/style.scss:
```diff
@@ -0,0 +1,82 @@
+  position: relative;
+}
+
+.image__top--center {
```
you can write instead `&--center` in 45 line

MHase commented on styles/style.scss:
```diff
@@ -0,0 +1,82 @@
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  padding-left: 25%;
```
to all `%` and `vh` `vw` elements in your styling. you don't have to use this type of measuring, because it's impossible to maintain true responsiveness, you can use pixels ;) the same goes for fonts where `em` was useable some time ago, but now mobile phone has resolution full HD or even 4K!

MHase commented on styles/style.scss:
```diff
@@ -0,0 +1,82 @@
+    }
+}
+
+.image__line--center {
```
and yea :D pseudoelement plx

MHase commented:
<img width="516" alt="Screenshot 2019-05-15 at 11 06 23" src="https://user-images.githubusercontent.com/11634131/57763348-7e59d580-7701-11e9-8d5a-718a5e5dfe2c.png">
🤔 

while you create branches etc. please name it properly 😞 `hero-maciek` is not a good name

Smutnyyy commented:
I've corrected my hero part. 
I've changed the line below the text - now it is shown as ::after psudoelement :)
The main section with image and the title "Design a Career You Love" is now called "hero". The next sections will be named "section--dark" and "section--light".

I've also deleted the _em, %, vh_, and replaced them by _px_.

That's all for now. I am waiting for next comments :) if everything is ok, I can write next part of the code :)

MHase commented on hero.html:
```diff
@@ -0,0 +1,20 @@
+  <body>
+    <section class="hero">
+      <div class="hero__image">
+        <div class="hero__image--center">
```
it looks like it's another variation of `hero__image` component, but instead it's just a wrapper for text 🤔 I think it's better to change naming here, because it's misleading

Smutnyyy replied:
Yes that's true. It was misleading however there was an image and a text. I corrected that to "hero__image-title"

MHase commented on style.scss:
```diff
@@ -0,0 +1,77 @@
+      background-color: $c_meteor;
+      right: 0;
+    }
+}
```
well... why did you create new file instead of editing the old one? 😅

MHase replied:
just to be sure, we want to create only desktop version of this page for now? because it's not responsive 😊

Smutnyyy replied:
I know that it is not responsive :( For now I want to create only desktop version. Maybe later I will add some media-querries to make that site responsive.

MHase commented on style.scss:
```diff
@@ -0,0 +1,77 @@
+  margin: auto;
+
+    & h1 {
+    width: 600px;
```
try not to set static `width` or `height` on elements like this :D you can set it on the wrapper instead and also it's better to set 'max-width', because if you will change the text to... let's say `Love` you will end with 600px component 🍠

Smutnyyy replied:
Is that rule (with not using static width or height) concerns all width or height? For example when I want to set the height of the main div. Then should I also use max- or min-widh, or then I can use width and set the static value.

I've changed the width in class "hero_title-center", but I left the static value in the other divs.

MHase commented on style.scss:
```diff
@@ -0,0 +1,77 @@
+    display: flex;
+    flex-direction: column;
+    height: 800px;
+  }
```
if I'm right you can move these properties to `hero__image` and remove completely `hero__image--center` wrapper 😊

Smutnyyy replied:
You're right. Corrected :)

MHase commented on hero.html:
```diff
@@ -0,0 +1,20 @@
+      <div class="hero__image">
+        <div class="hero__image--center">
+          <div class="hero__text--center">
+            <h1>Design a Career You Love</h1>
```
you can add `class="hero__text--center"` to `h1` tag and remove `div` wrapper :)

Smutnyyy replied:
div wrppaer removed.

MHase commented on style.scss:
```diff
@@ -0,0 +1,77 @@
+}
+
+.hero__image {
+    height: 800px;
```
try to achieve this height with paddings in component's content, because if you will add something you will have to change `height` every time

Smutnyyy replied:
Don't understand, how I can add paddings to achieve that height. The height of that image has to be 800px. When I want to add sth new I will cover the image and that's all. The height will not change (if I understand that correctly)

MHase replied:
I don't think this hero section has to be exactly 800px, it should be more like a background, but this question is for a designer, so I think we can leave it as it is

MHase commented on hero.html:
```diff
@@ -0,0 +1,20 @@
+    <link rel="stylesheet" type="text/css" href="colors.css">
+  </head>
+  <body>
+    <section class="hero">
```
you can set background image in hero itself, you don't have to create individual component for and image

MHase replied:
🤔 I don't quite understand why you have created strcuture like this:
```
<section class="hero">
      <div class="hero__image-title">
          <h1 class="hero__title--center">Design a Career You Love</h1>
      </div>
    </section>
```

and assigned background image to `hero__image-title` instaed of `hero`, because with additional div you have created unnecessary container 🤔

MHase commented:
you can easily proceed to next part of the project, but DO NOT mix different components in one PR, because it will be hard to read the code/changes.
merge this PR to master and then create another PR with grid elements instead of pushing everything on one branch :)
